src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryParser.java

```diff
@@ -183,7 +183,7 @@ public Filter parse(QueryParseContext parseContext) throws IOException, QueryPar
         }
```

Comment:
It did not cause errors because Filter now extends Query in Lucene but I agree it's cleaner.